book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,201 @@
+# Run a Wallaroo Application in Docker
+
+In this section, we're going to run an example Wallaroo application in Docker. By the time you are finished, you'll have validated that your Docker environment is set up and working correctly. If you haven't already completed the Docker setup [instructions](book/getting-started/docker-setup.md), please do so before continuing.
+
+There's a couple Wallaroo support applications that you'll be interacting with for the first time:
+
+- Our Metrics UI that allows you to monitor the performance and health of your applications.
+- Giles receiver is designed to capture TCP output from Wallaroo applications.
+- Giles sender is used to send test data into Wallaroo applications over TCP.
+- Machida, our program for running Wallaroo Python applications.
+
+You're going to set up our "Celsius to Fahrenheit" example application. Giles sender will be used to pump data into the application. Giles receiver will receive the output and our Metrics UI will be running so you can observe the overall performance.
```

Comment:
`We will set up the "Celsius to Fahrenheit" example application. We will use Giles Receiver for the sink, Giles Sender to send input data to the source, and the Metrics UI to observe the overall performance.`